src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/task/VisitTask.kt

```diff
@@ -18,6 +19,10 @@ class VisitTask(
   }
 
   @Scheduled(cron = "\${task.expired-visit.cron:0 0/15 * * * ?}")
+  @SchedulerLock(
+    name = "deleteExpiredVisitsTask",
+    lockAtLeastFor = "PT5M", lockAtMostFor = "PT10M"
```

Comment:
is 10mins too much,... might be reading this incorrectly

Reply:
Should I reduce it to 5?

Reply:
sure

Reply:
I have now updated the lock (atLeastFor and AtMostFor) to 5 mins.